sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/CliSuite.scala

```diff
@@ -454,24 +454,24 @@ class CliSuite extends SparkFunSuite with BeforeAndAfterAll with BeforeAndAfterE
   }
 
   test("SPARK-30049 Should not complain for quotes in commented lines") {
-    runCliWithin(1.minute)(
+    runCliWithin(3.minute)(
       """SELECT concat('test', 'comment') -- someone's comment here
         |;""".stripMargin -> "testcomment"
     )
+    runCliWithin(3.minute)(
+      """SELECT concat('test', 'comment'),
+        |    -- someone's comment here
+        | 2;""".stripMargin -> "testcomment"
+    )
```

Comment:
Could you move this test into a new test unit just like `test("SPARK-31102: XXXX") {...}`?